packages/core/src/components/Dropdown/__tests__/__snapshots__/Dropdown.snapshot.test.js.snap

```diff
@@ -193,18 +193,18 @@ exports[`Dropdown renders correctly snapshot driver should open menu on focus if
     id="dropdown-menu-id"
   >
     <span
+      aria-atomic="false"
       aria-live="polite"
-      class="css-1laao21-a11yText"
+      aria-relevant="additions text"
+      class="css-1f43avz-a11yText-A11yText"
     >
       <span
-        id="aria-selection-event"
-      >
-         
-      </span>
+        id="aria-selection"
+      />
       <span
         id="aria-context"
       >
-           0 results available.   Selected:  is focused ,type to refine list, press Down to open the menu, 
+            Selected:  is focused ,type to refine list, press Down to open the menu, 
```

Comment:
It seem like it was a bug at react select. no reason that 0 results available would be read to user here, as the input does not filter the results at all. it should only appear if the input filter does not match any option. in here all the options are available

Reply:
It's not breaking, right?

Reply:
it is only for the screen reader, and it is a bug fix, so don't think so